content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,506 @@
+<!--more-->
+### Table of Contents
+1. Prerequisites.
+2. Goals of the tutorial.
+3. Understanding hypothesis testing.
+4. Confidence of null hypothesis.
+5. Statistical techniques for hypothesis testing.
+6. Chi-square test - Python implementation.
+7. T-Test - Python implementation.
+8. ANOVA - Python implementation.
+9. Conclusion.
+10. Additional resources.
+
+### 1. [Prerequisites](#1-prerequisites)
+The reader of this tutorial should have basic knowledge of python programming, must have an idea of the concept of exploratory data analysis and how pandas and numpy work, must have elementary understanding of statistics, and must be a data science enthusiast.
+
+### 2. [Goal of the Tutorial](#2-goal-of-the-tutorial)
+At the end of this tutorial, readers should be able to 
+- understand clearly the concept of statistical hypothesis testing.
+- perform the implementation of t-test, chi-squared test & ANOVA with python and a new dataset.
+- differentiate clearly the trade-off between type 1 and type 2 error of null hypothesis.
+- explain the confidence of the null hypothesis.
+- play with basic pandas function for exploratory data analysis.
+- figure out the best case-study for "t-test", "chi-squared test" and "ANOVA test".
+
+### 3. [Understanding Hypothesis Testing](#3-understanding-hypothesis-testing)
+There are two types of hypothesis testing, namely, null hypothesis and alternate hypothesis.
+
+[**Null hypothesis**](https://en.wikipedia.org/wiki/Null_Hypothesis:_The_Journal_of_Unlikely_Science) is the initial assumption about an event (also referred to as the ground truth), whereas [**alternate hypothesis**](https://en.wikipedia.org/wiki/Alternative_hypothesis) is an assumption that counters the initial assumption". The null hypothesis is a statement believed to be true unless we can prove otherwise beyond a reasonable doubt.
+
+Consider a situation where a seventy-year-old woman has a visible bump in her belly. The medical opinion is that the bump could be a fibroid. In this case, our initial conclusion (or our null hypothesis) is that -
+
+**"She has fibroid"**.
+while our alternate hypothesis will be
+**"She does not have fibroid"**.
+
+In trying to make a valid conclusion between two assumptions on a certain sample of data, we will refer to the null hypothesis (or our initial assumption) as H0 hypothesis and the alternate hypothesis (or the counter assumption) as H1 hypothesis.
+
+To carry out hypothesis testing, the first step is to form and label the null hypothesis as H0 and the alternate hypothesis as H1. The next step is to collect all data samples available to support the null hypothesis, collect data pertaining to the hypothesis and analyze it to make a decision if H0 can be accepted or rejected.
+ 
+While doing that, there is a likelihood for four events to happen.
+There are four possibilities to be considered here:
```

Comment:
This could be edited as -
`While doing that, there is a likelihood for four events to happen:`
The second sentence can be removed.